core/src/util/extrude.cpp

```diff
@@ -0,0 +1,82 @@
+#include "extrude.h"
+#include <cmath>
+#include <cstdlib>
+
+namespace Tangram {
+
+Extrude parseExtrudeString(const std::string& _str) {
+
+    // Values specified from the stylesheet are assumed to be meters with no unit suffix
+    float first = 0, second = 0;
+
+    if (_str == "true") {
+        // "true" means use default properties for both heights, we indicate this with NANs
+        return Extrude(NAN, NAN);
+    }
+
+    if (_str == "false") {
+        // "false" means perform no extrusion
+        return Extrude(0, 0);
+    }
+
+    // Parse the first of two possible numbers
+    const char* pos = _str.c_str();
+    char* end = nullptr;
+
+    // Get a float if possible & advance pos to the end of the number
+    first = std::strtof(pos, &end);
+    if (pos == end) {
+        // No numbers found, return zero extrusion
+        return Extrude(0, 0);
+    }
+
+    // Advance and skip the delimiter character
+    pos = end + 1;
```

Comment:
From the docs (https://mapzen.com/documentation/tangram/draw/#extrude), it seems that the extrude parameter can have `m`, `px` units associated with the value, defaulting to `m` by default. On stop offset interpolation I just added some allowed unit check based on the draw key https://github.com/tangrams/tangram-es/blob/interpolate-offsets/core/src/scene/styleParam.cpp#L56_L60, we can probably review and merge #429, then parse and add checks for allowed units for sanity here.